core/runtimeservices/src/main/java/org/apache/causeway/core/runtimeservices/publish/EntityPropertyChangePublisherDefault.java

```diff
@@ -66,6 +67,9 @@ public class EntityPropertyChangePublisherDefault implements EntityPropertyChang
 
     private Can<EntityPropertyChangeSubscriber> enabledSubscribers = Can.empty();
 
+    @Value("${entity.property.change.publisher.bulk.threshold:1}")
```

Comment:
This property name needs to change, and also should be read from `CausewayConfiguration` so that it is typesafe.

The property name should be `causeway.core.runtime-services.entity-property-change-publisher.bulk.threshold`

Reply:
Fixed